index.html

```diff
@@ -42,6 +41,133 @@ <h1><span class="color-blue"> Hey there. <br> I'm Ale. </span><br><span id="titl
+                            <li>HTML</li>
+                        </ul>
+                    </section>
+                    <button>See project</button>
+                </div>   
+            </section>
+            <section class="work n5">
+                <div class="white-area">
+                    <h4>Multi-Post Stories Gain+Glory</h4>
+                    <section class="project-lang">
+                        <ul class="project-lang-list">
+                            <li>Ruby on Rails</li>
+                            <li>CSS</li>
+                            <li>JavaScript</li>
+                            <li>HTML</li>
+                        </ul>
+                    </section>
+                    <button>See project</button>
+                </div>   
+            </section>
+            <section class="work n6">
+                <div class="white-area">
+                    <h4>Multi-Post Stories Gain+Glory</h4>
+                    <section class="project-lang">
+                        <ul class="project-lang-list">
+                            <li>Ruby on Rails</li>
+                            <li>CSS</li>
+                            <li>JavaScript</li>
+                            <li>HTML</li>
+                        </ul>
+                    </section>
+                    <button>See project</button>
+                </div>   
+            </section>
+        </section>
+        <section class="about-myself">
+            <div class="text">
+                <h1 class="color-blue">About me</h1>
+                <p class="color-blue"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hesitate to contact me.
+                </p>
+                <button>Get my resume</button>
+            </div>
+            <section class="languages">
+                <h5 class="lang">Languages</h5>
+                <ul  class="lang-list">
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                </ul>
+            </section>
+            <section class="framework">
+                <h5 class="lang">Frameworks</h5>
+                <ul  class="lang-list">
+                    <li>React.js</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSpec</li>
+                    <li>Capybara</li>
+                    <li>Selenium</li>
+                </ul>
+            </section>
+            <section class="skills">
+                <h5 class="lang">Skills</h5>
+                <ul  class="lang-list">
+                    <li>Database Management</li>
+                    <li>Version Control</li>
+                    <li>CLI</li>
+                    <li>Web Development</li>
+                    <li>API Design</li>
+                </ul>
+            </section>            
```

Comment:
- [ ] Aslo for this section, following the best practices, the category list should be in a ul tag with each item being an li, 
`
                        <ul class="languages">
                <li class="lang">Languages</li>
                    <ul  class="lang-list">
                        <li>JavaScript</li>
                        <li>Ruby</li>
                        <li>HTML</li>
                        <li>CSS</li>
                    </ul>
                <li class="lang">Frameworks</li>
                    <ul  class="lang-list">
                        <li>React.js</li>
                        <li>Ruby on Rails</li>
                        <li>RSpec</li>
                        <li>Capybara</li>
                        <li>Selenium</li>
                    </ul>
                <li class="lang">Skills</li>
                    <ul  class="lang-list">
                        <li>Database Management</li>
                        <li>Version Control</li>
                        <li>CLI</li>
                        <li>Web Development</li>
                        <li>API Design</li>
                    </ul>
            </ul>   
`

- - THis should be the structure of this section, I'll suggest you update this section. :+1: :100: